client/src/components/select/index.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react';
+import '../../App.css';
+import './select.css';
+
+class Select extends Component {
+  render() {
+    const { labelText, firstItem } = this.props;
+    return (
+      <div className='container-select'>
+        <label className='label-style'>{labelText}</label>
+        <select className='select-style'>
```

Comment:
What happens if I select something from this option list?

Reply:
Its mock data at this moment,
we  will get the data from api for Nigerian Cities

Reply:
Put an object that represents your mock data in your state, and then handle this object in your jsx. So that when you've got a server, you don't need to rewrite this code again. This goes on to the rest of the comments @Balsam-Faysal .

Reply:
Done